types/heremaps/index.d.ts

```diff
@@ -5681,7 +5694,7 @@ declare namespace H {
              * @param opt_locale {(H.ui.i18n.Localization | string)=} - the language to use (or a full localization object).
              * @returns {H.ui.UI} - the UI instance configured with the default controls
              */
-            static createDefault(map: H.Map, mapTypes: H.service.Platform.MapTypes, opt_locale?: H.ui.i18n.Localization | string): UI;
+            static createDefault(map: H.Map, mapTypes: H.service.MapType | H.service.DefaultLayers, opt_locale?: H.ui.i18n.Localization | string): H.ui.UI;
```

Comment:
The reason for this change is that I noticed the [documentation](https://developer.here.com/javascript-apis/documentation/v3/maps/topics_api/h-ui-ui.html#h-ui-ui__createdefault-static) is different from the existing types. However the documentation isn't very good and only states `mapTypes:
{Object<H.service.MapType>}`. 
However I changed it so that we have `mapTypes: H.service.Platform.MapTypes | H.service.MapType | H.service.DefaultLayers` now which shouldn't break any old implementations.